### ISSUE TITLE
RNTextDetector detectFromUri doesn't return detected text

### ISSUE THREAD

**Issue by dreadful-dev** [zsajjad/react-native-text-detector#11]
I've managed to get this library built and running on my device and  I have my tessdata folder referenced correctly as described in the readme. `RNTextDetector.detectFromUri` seems to be detecting text, however the detect text isn't provided in the `visionResp`, only the bounding and position data for the detected text. Is there something else I need to do to get the text?

I've implemented the usage of `RNTextDetector.detectFromUri` based off of the BusinessCard sample project.

https://github.com/zsajjad/BusinessCard/blob/master/app/App.js#L98

Here are screenshots of the app and what is logged to the console.
![image](https://user-images.githubusercontent.com/34455148/52291626-c322e300-2940-11e9-81c7-aa333eaa7751.png)

![image](https://user-images.githubusercontent.com/34455148/52291309-16e0fc80-2940-11e9-90f1-dcfd7d21836b.png)

edit: I've also tried getting much clearer photos than the example, and that hasn't worked either.

**Comment by dreadful-dev** [comment]
Figured it out, i needed to select "copy to destination if needed" when referencing the tessdata folder

**Comment by vaiden** [comment]
In plain English: @dreadful-dev means that you should follow [the this part of the installation guide](https://github.com/zsajjad/react-native-text-detector#attach-tesseract-languages-you-want-to-use-in-your-app), even if you've linked the lib automatically.
